coreneuron/nrniv/profiler_interface.h

```diff
@@ -260,6 +260,17 @@ struct NullInstrumentor {
 
 }  // namespace detail
 
+namespace Instrumentor {
+    struct phase {
+        phase() {
+            detail::Instrumentor::phase_begin();
```

Comment:
where do you capture the argument? (if this is instantiated as `p("checkpoint"`)). As per original semantic it was supposed to call `phase_begin("checkpoint")`.

Reply:
Implemented 2e743e4.